app/views/mortgage_calculator/stamp_duties/show.html.erb

```diff
@@ -1,17 +1,18 @@
 <div ng-controller="CalculatorCtrl" class="mortgagecalc__container stamp-duty">
-  <%= render 'header' %>
 
   <% @stamp_duty.errors.full_messages.each do |m| %>
     <span style="color:red;"><%= m %></span>
   <% end %> 
-  <h2 class="intro stamp-duty__subheading"><%= I18n.t('stamp_duty.title') %></h2>
   <div wizard on-finish="finishedWizard()" hide-indicators='true'>
     <div wz-step class="ng-hide">
+      <h1 class="stamp-duty__heading"><%= I18n.t('stamp_duty.h1') %></h1>
```

Comment:
Can we change the name of the locale key h1 to heading, please? IMHO, the locale should not know about html tags. Wdyt?

Reply:
Agreed. I'll change the key name.

Reply:
@tomas-stefano updated